recipe/meta.yaml

```
@@ -1,50 +1,83 @@
 {% set name = "dataframe_image" %}
-{% set version = "0.1.1" %}
-
+{% set version = "0.2.7" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 8d6739da53c66bd557f217237bab45943a452760b808bb7a3e635a939eed0bb5
+  url: https://github.com/dexplo/dataframe_image/archive/v{{ version }}.tar.gz
+  sha256: 6d96368cbf64e8291587242bd6442ed2961cc021a06d2ccf1d309fe460e77831
 
 build:
   number: 0
-  noarch: python
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
   entry_points:
     - dataframe_image=dataframe_image._command_line:main
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  # nbconvert is not available for py313
+  skip: True  # [py<38 or py>312]
```

```suggestion
  skip: True  # [py<38]
```
nbconvert is not available because not built yet, but that doesn't mean this recipe won't be compatible once it is.
It is less maintenance to just remove the configuration in abs.yaml. (That is the whole point of that configuration btw, avoiding adding unnecessary skip statements to recipes.)